### PR TITLE
Korsin/clearing document name

### PR DESCRIPTION
**Pull request by KorSin**
> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to? (*Refer to issue here*)
> * How is it solving the issue?
> * Did you add or update any new dependencies that are required for your change?

@neubs-bsi 
@oheger-bosch 

*Type of change*:  bug fix

### Checklist
Must:
- [x] All related issues are referenced in commit messages

Optional: *(delete if not applicable)*
- [x] I have provided tests for the changes (if there are changes that need additional tests)

### CODE REVIEW

File: assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/SW360UpdaterTest.java [hunk 1]
```diff
@@ -61,38 +58,21 @@ public void testClearingReportGeneratorMustNotBeNull() {
...
+                .thenThrow(new SW360ClientException("Clearing doc generation error"));
         SW360Updater updater = new SW360Updater(updaterImpl, configurationMock, clearingReportGenerator);
 
-        // and an exception is nevertheless thrown by SW360ReleaseClientAdapterAsyncImpl
         Assertions.assertThatThrownBy(updater::execute)
                 .isInstanceOf(SW360ClientException.class);
 
-        // but we expect that attachment upload worked one time exactly
-        verify(releaseClientAdapter, times(1)).uploadAttachments(any());
+        verify(updaterImpl, never()).artifactToReleaseWithUploads(any(), any(), anyMap());
```
Comment (oheger-bosch) [inline assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/SW360UpdaterTest.java:75]: One major goal of this test was to verify whether exceptions thrown during the processing of an artifact are caught and do not cause the process to abort. Therefore, the clearing report generator was configured to throw an error once and then succeed. In the second iteration, a successful upload should be performed.

With the reworked test, this is no longer the case. Was there a reason to strip down the test in this way?

Comment (KorSin) [reply assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/SW360UpdaterTest.java:75]: From the updater, I removed the, in my opinion, redundant second upload of attachments. But with that the method `uploadAttachments` is never called by the `SW360Updater` anymore. Consequently, I removed this from the test and changed it to verify that the method `artifactToReleaseWithUploads` is never called. Furthermore, I removed other mocking calls that are not called anymore.

Comment (KorSin) [reply assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/SW360UpdaterTest.java]: After a discussion with @nnobelis I finally understood the intention of the test. Please have a look at the changes.

**Review by oheger-bosch — APPROVED** [review]
Looks good now. Thank you!
I will merge when CI is green.

**Pull request merged**